lib/pos_mempool_test.go

```diff
@@ -198,6 +198,7 @@ func _blockchainSetup(t *testing.T) (_params *DeSoParams, _db *badger.DB) {
 	require := require.New(t)
 
 	chain, params, db := NewLowDifficultyBlockchain(t)
+	params.ForkHeights.BalanceModelBlockHeight = 1
```

Comment:
Why are we putting this in here? jw

Reply:
The `CheckTransactionSanity` call that I added to `validateTransaction` needs to think we're on balance model. Otherwise, it looks for txinputs/outputs.